third_party/pyth/price-service/src/index.ts

```diff
@@ -61,6 +61,7 @@ async function run() {
   const wsAPI = new WebSocketAPI(listener, promClient);
 
   listener.run();
+  listener.removeExpiredValuesFromVaasCache();
```

Comment:
The name suggests its a one time action, maybe `runCacheCleanupLoop` or something similar?